src/interaction/FENECapped.py

```diff
@@ -24,64 +24,65 @@
+
+The capped FENE potential is employed as:
+
 .. math::
 
 	U = -\frac{1}{2}r_{max}^2  K \cdot
- 				 log\left(1 - \frac{D - r_{0}}{r_{max}}^2\right)
+                 log \left[1 - \left(\frac{D - r_0}{r_{max}}\right)^2\right],
 
-where :math:`D = dist` if 
+where
+
+.. math::
 
-:math:`{cap_{rad}}^2>dist` 
+    D = min \left(r, r_{\it{cap}}\right).
 
-and :math:`D = cap_{rad}` else.
+.. py:class:: espressopp.interaction.FENECapped(K = 30.0, r0 = 0.0, rMax = 1.5, cutoff = inf, r_cap = 1.0, shift = 0.0)
 
+    :param real K: attractive force strength (in :math:`\epsilon / \sigma^2` units)
+    :param real r0: displacement parameter (in :math:`sigma` units)
+    :param real rMax: size parameter (in :math:`sigma` units)
+    :param real cutoff: cutoff radius
+    :param real r_cap: radius of capping (in :math:`sigma` units)
+    :param real shift: shift of the potential
 
 
+After setting up the potential you have to apply it to the particles in the pair list (bondlist):
 
+.. py:class:: espressopp.interaction.FixedPairListFENECapped(system, bondlist, potential)
 
+    :param object system: system object :func:`espressopp.System`
+    :param list bondlist: list of bonds :func:`espressopp.FixedPairList`
+    :param object potential: bonded potential, in this case :func:`espressopp.interaction.FENECapped`
 
+    **Methods**
 
-.. function:: espressopp.interaction.FENECapped(K, r0, rMax, cutoff, caprad, shift)
+    .. py:method:: getFixedPairList()
 
-		:param K: (default: 1.0)
-		:param r0: (default: 0.0)
-		:param rMax: (default: 1.0)
-		:param cutoff: (default: infinity)
-		:param caprad: (default: 1.0)
-		:param shift: (default: 0.0)
-		:type K: real
-		:type r0: real
-		:type rMax: real
-		:type cutoff: 
-		:type caprad: real
-		:type shift: real
+        :rtype: A Python list of pairs (the bondlist)
 
-.. function:: espressopp.interaction.FixedPairListFENECapped(system, vl, potential)
+    .. py:method:: getPotential()
 
-		:param system: 
-		:param vl: 
-		:param potential: 
-		:type system: 
-		:type vl: 
-		:type potential: 
+        :rtype: potential object
 
-.. function:: espressopp.interaction.FixedPairListFENECapped.getFixedPairList()
+    .. py:method:: setFixedPairList(bondlist)
 
-		:rtype: A Python list of lists.
+        :param list bondlist: fixed-pair list (bondlist)
 
-.. function:: espressopp.interaction.FixedPairListFENECapped.getPotential()
+    .. py:method:: setPotential(potential)
 
-		:rtype: 
+        :param object potential: a potential applied to all pairs in the bondlist
 
-.. function:: espressopp.interaction.FixedPairListFENECapped.setFixedPairList(fixedpairlist)
+**Example of usage**
 
-		:param fixedpairlist: 
-		:type fixedpairlist: 
+>>> Please, refer to the example of FENE potential
 
-.. function:: espressopp.interaction.FixedPairListFENECapped.setPotential(potential)
+Go to FENE-example :py:class:`espressopp.interaction.FENE`
```

Comment:
what is FENE-example?

Reply:
it refers to the example on FENE-potential usage. Should I change the formulation here?

Reply:
Does `:py:class:`espressopp.interaction.FENE` convert to a link ?

Reply:
Good :+1: